roslibrust/Cargo.toml

```diff
@@ -25,15 +27,23 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 smart-default = "0.6"
 thiserror = "1.0"
-tokio = { version = "1.20", features = ["net", "macros", "time", "rt-multi-thread"] }
```

Comment:
Do you have a different toml linter than me? Does rustfmt hit the toml files? Just want to get us on same page.

Reply:
I will have to check what vscode extension I have, it very rarely changes the file, but pretty silly of us to keep changing it back and forth.

Reply:
I'm using Even Better TOML: https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml